README.md

```diff
@@ -2,71 +2,64 @@
 
-Further runtime documentation can be found in [Rive's help center](https://help.rive.app/runtimes).
+A React Native runtime library for [Rive](https://rive.app).
 
-## Installation
+This library is a wrapper around the iOS/Android runtime, providing a component and ref pattern for React Native applications.
 
-- install `rive-react-native` dependency using yarn or npm
+## Table of contents
 
-```sh
-npm install rive-react-native
-```
+- :star: [Rive Overview](#rive-overview)
+- 🚀 [Getting Started & API docs](#getting-started)
+- :mag: [Supported Devices](#supported-devices)
+- :books: [Examples](#examples)
+- 👨‍💻 [Contributing](#contributing)
+- :question: [Issues](#issues)
 
-or
+## Rive Overview
 
-```sh
-yarn add rive-react-native
-```
+[Rive](https://rive.app) is a real-time interactive design and animation tool that helps teams create and run interactive animations anywhere. Designers and developers use our collaborative editor to create motion graphics that respond to different states and user inputs. Our lightweight open-source runtime libraries allow them to load their animations into apps, games, and websites.
 
-- install pods for your ios project. Go to the project `ios` directory and run
+:house_with_garden: [Homepage](https://rive.app/)
 
-```sh
-pod install
-```
+:blue_book: [General help docs](https://help.rive.app/)
 
-- Add empty swift file in order to create `Bridging-Header` file if it doesn't exist. _(optional)_
+🛠 [Resources for building in Rive](https://rive.app/resources/)
 
-- add this dependency to your project inside `android/app/build.gradle`
+## Getting Started
 
-```groovy
-dependencies {
-  implementation "androidx.startup:startup-runtime:1.0.0"
-}
-```
+Follow along with the link below for a quick start in getting Rive React Native integrated into your multi-platform applications.
 
-More info [here](https://github.com/rive-app/rive-android#manually-initializing-rive).
+[Getting Started with Rive in React Native](https://help.rive.app/runtimes/overview/react-native)
 
-Put `.riv` files inside your project:
+[API documentation](https://help.rive.app/runtimes/overview/react-native/props)
 
-- `raw` directory on Android
-- `Assets` directory on iOS
+## Supported Devices
 
-## Usage
+Because this runtime library has a dependency on the [Rive Android](https://github.com/rive-app/rive-android) and [Rive iOS](https://github.com/rive-app/rive-ios) runtimes, the supported devices align with each of these dependencies minimum supported devices, as well as the minimum device requirements of the React Native framework.
 
-```tsx
-import Rive from 'rive-react-native';
+- iOS: **14.0+**
+- Android:
+  - Minimum SDK version: **21**
+  - Target SDK version: **29**
```

Comment:
@umberto-sonnino @mjtalbot wanted to double check this is right here..as I mainly looked to the following places:
https://github.com/rive-app/rive-react-native/blob/main/android/build.gradle#L34

https://github.com/rive-app/rive-react-native/blob/main/android/gradle.properties#L4

Reply:
I think we've updated it to be target 31 (which is the latest) -- [here's](https://github.com/rive-app/rive-android/blob/master/kotlin/build.gradle) the latest runtime build.gradle

Reply:
Ah! ok, ill need to update this project's build.gradle too then